doc/release-notes.md

```diff
@@ -43,8 +43,45 @@ a reindex or re-sync the whole chain.
 Notable changes
 ===============
 
-TODO
-----
+Fix respends of freshly received InstantSend transactions
+---------------------------------------------------------
+
+A bug caused respends to not work before a received InstantSend transaction was confirmed in at least
```

Comment:
Should probably clarify that this bug affects transactions received in Core _wallet_ only, other wallets should do just fine cause on protocol level/network-wide chained InstantSend works as designed.